pkg/dumper/postgres/dumper.go

```diff
@@ -47,6 +47,15 @@ func (d *pgDumper) DumpStructure(sql string) error {
 	return nil
 }
 
+// quoteIfNotQuoted this function avoid quote strings that are already quoted.
+func quoteIfNotQuoted(s string) string {
+	if len(s) > 0 && s[0] == '"' {
+		return s
+	}
+
+	return strconv.Quote(s)
+}
```

Comment:
to ensure the double quotes, maybe we could `strings.Trim`  and leverage the `%q` formatting instead where is needed

Reply:
Thanks for the feedback! For sure, I can update the code.

Just to make sure, are you suggesting we don't use the function and do something like:

```
query := fmt.Sprintf("ALTER TABLE %q ADD CONSTRAINT %q %s", strings.Trim(fk.tableName), strings.Trim(fk.constraintName), fk.constraintDefinition)
```

Reply:
yes, the `strings.Trim` take `"` as an argument too, but you got it :wink: